packages/liblzma/liblzma.spec

```diff
@@ -0,0 +1,49 @@
+Name: %{_cross_os}liblzma
+Version: 5.2.5
+Release: 1%{?dist}
+Summary: Library for XZ and LZMA compressed files
+URL: https://tukaani.org/xz
+License: Public Domain
```

Comment:
Our standard for the `License` field is an SPDX license identifier, and obviously there's not one on the official list in this case.

For the microcode package, I went with a scancode LicenseRef. It looks like maybe [LicenseRef-scancode-lzma-sdk-pd](https://scancode-licensedb.aboutcode.org/lzma-sdk-pd.html) could work here but I'll need to check.

Reply:
Thanks, I had indeed missed that the other third-party packages use SPDX license identifiers.